### PR TITLE
Margins and Padding for Media Player,Related Content, Top Story,Paragraph

### LINKED ISSUE #8305: Emotion - UX bugs to fix post-merge
**Describe the bug**
UX issues following the Emotion migration that can be fixed after merge

### UX issues found following review

1. Paragraph text on STYs needs 40px additional space to the right above 1008px to reduce the width the text spans.

The reason for it being like this is for reading comfort - once you go beyond a certain number of characters on a line it becomes more difficult to read

Left- _designs_, Right - _live_

Screenshot 2020-11-04 at 17 38 47

2. 240 - 399px there should be a 0.5rem margin above the media player (as there is at higher breakpoints)
Screenshot 2020-11-04 at 13 21 17

3. Currently no bottom padding below 'Related content' across break points, and no bottom margin below 600px. 

Prior to migration - it had 1.5rem padding and 2rem margin above 600px (3.5rem total), and 2rem padding and 1rem margin (3rem total) below 600px. This is the problem area:
Screenshot 2020-11-09 at 09 53 13

4. L/R padding on radio schedule below 600px that shouldn't be there - captured in a Psammead issue https://github.com/bbc/psammead/issues/3963

5. The 'Section' within 'Top story' has a 1.5rem margin-bottom on live, above 1008px. After the emotion work this has been removed. We have been applying the margin to the wrapping div - which should have an additional 1.5rem added to the margin-bottom above the 1008px break point.
Screenshot 2020-11-09 at 10 12 18

### PR DESCRIPTION
**Pull request by emilysaffron**
Resolves #8305

**Overall change:**
- Correct padding for paragraph text on STYs
- Correct margin added to the top of media player
- Correct padding added to related content
- Correct padding added to the top of story section

**Code changes:**

- StyledParagraph added in Paragraph with a media query for a min of 1008px when 40px right padding gets applied
- Media query added which applies 0.5rem top margin up to 399px
- Media queries added on LegacyGridItemLarge which applies 1.5rem padding and 2rem margin above 600px and 2rem 
   padding and 1rem margin below 600px. 
- Media query added to ResponsiveComponentWrapper to add 1.5rem margin bottom above 1008px

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by ryanmccombe** [comment]
Hi Emily

Some weirdness going on in this branch it seems

It looks like it is re-introducing the code that was removed as part of this PR: https://github.com/bbc/simorgh/pull/8431 - you can see a lot of “recent episodes” stuff in your files changed tab

This seems to be because this commit is not being recognised as a merge commit:  https://github.com/bbc/simorgh/pull/8366/commits/02756bdd5f666ab1ca689e8430a8e85ebdc03154

It is being recognised as an intentional change

You can check if a commit is a merge commit using `git cat-file -p your-commit-id` eg:
Screen Shot 2020-11-23 at 14 13 50

Any merge commit should have multiple parents, like this one:

Screen Shot 2020-11-23 at 14 14 10

The easiest solution at this point would probably be to just remove all the recent episode stuff again, using that diff as a reference: https://github.com/bbc/simorgh/pull/8431

You can reset a file to how it is on latest using `git checkout origin/latest your-file-name`

Let me know if you need any help :)

**Review by ryanmccombe — APPROVED** [review]
👍

File: src/app/containers/Paragraph/index.jsx [hunk 1]
```diff
@@ -10,14 +12,20 @@ import { GridItemMedium } from '#app/components/Grid';
 
 const componentsToRender = { fragment, urlLink: InlineLink, inline: Inline };
 
+const StyledParagraph = styled(Paragraph)`
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    padding-right: 40px;
```
Comment (rebeccamcginn) [inline src/app/containers/Paragraph/index.jsx]:
```suggestion
    padding-right: ${GEL_SPACING_QUIN};
```

**Review by rebeccamcginn — APPROVED** [review]
🥇 Just need to update the snapshots after that commit :)

**Pull request merged**